app/scripts/views/settings/clients.js

```diff
@@ -66,6 +66,12 @@ define(function (require, exports, module) {
 
     context () {
       let clients = this._attachedClients.toJSON();
+      clients.forEach(function (client) {
+        client.title = client.name;
```

Comment:
@divyabiyani if we go with this approach let's use the existing `_formatAccessTime` function that already goes through all the items. We would need to rename that function though because it won't be just formatting the access time anymore.